releasenotes/2.1.1.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: releasenotes
+title: Release Notes for Version 2.1.1
+date: 2018-08-09 16:00:00
+release_notes: true
+permalink: /releasenotes/2.1.1/
+---
+
+1. [Introduction](#intro)
+1. [Release Highlights](#highlights)
+1. [Known Issues](#knownissues)
+1. [Credits](#credits)
+1. [Test Results](#test)
+
+## <a id="intro"></a>Introduction
+
+Apache jclouds 2.1.1 is the first bugfix release after jclouds 2.1 and comes with several bugfixes and performance improvements. 
+You can [read the official announcement here](https://s.apache.org/jclouds211). To get started with jclouds, please see the [jclouds installation guide](/start/install/).
+
+## <a id="highlights"></a>Release Highlights
+
+Apache jclouds 2.1.1 new features include:
+
+* Add support for c5 and R4 instance types in AWS ec2.
+* Add the new Paris region to aws-ec2
+* Support different "domain" for "identity/user" and "project" in OpenStack Keystone V3
```

Comment:
`Support different "domain" values` or `"domain" settings` or so?

Reply:
Not sure what should be the writing here, @demobox @andreaturli ? Is there a reference for this new feature?

Reply:
Here you are: https://github.com/jclouds/jclouds/pull/1204
The contributor took the tie to put up a nice discussion on JIRA and to also open a PR adding the documentation to our site. Nice!